### ISSUE TITLE
Japanese and Chinese translations are not working

### ISSUE THREAD

**Issue by CvH** [xbmc/xbmc#15154]
### Describe the bug
After installing Japanese or Chinese language at Kodi no symbols are shown. This happens at LibreELEC and Windows (with installed and switched languages at Windows).

## Expected Behavior
Kodi should show chin/jap translations instead of nothing

## Actual Behavior
Showing just blocks instead of fonts.

## Possible Fix
install chin/jap enabled fonts via the language addons ?

### To Reproduce
install and activate Japanese as kodi interface language

### Debuglog
[kodi.log](https://github.com/xbmc/xbmc/files/2716342/kodi.log)

### Screenshots 
![image](https://user-images.githubusercontent.com/1355173/50537999-835e2400-0b68-11e9-9f0e-1153336a33b8.png)

### Your Environment
 - [ ] Android
 - [ ] iOS
 - [ x ] Linux
 - [ ] OSX
 - [ ] Raspberri-Pi
 - [ x ] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version: 18.0-RC4 Git:20181228

**Comment by pkerling** [comment]
Yeah, I also noticed that before, I think it's a very valid issue. Thanks for reporting. Workaround for now (if you didn't know already) is to set the font to "Arial based" *before* changing the language.

@phil65 I think this might be seen as skin-related issue since Estuary ships Noto Sans, can you comment here?

Possible solutions that I see:
* Ship a more complete font with Estuary - easy
* Fall back to the DejaVu Sans we ship for characters not present in the selected (skin) font (also maybe ship Noto Sans instead) - probably too complicated and far-reaching for v18

**Comment by pkerling** [comment]
Fall-back font is out for v18, too invasive (looked into it); but I think it's the proper fix for the roadmap

My personal suggestion would be to change our default-shipped font from DejaVu to Noto Sans with CJK support and remove Noto from Estuary

**Comment by pkerling** [comment]
> Shipping a catch all font is too big (9mb vs ~350kb)

We already ship DejaVu Sans anyway, which is 5 MB and supports CJK as far as I'm aware. Other additional languages that it might not cover (no idea if there are any) would not result in any significant increase. So this point does not really hold.

> why don't ship the font with the language addon instead

I don't think we have the infrastructure in place for autoswitching the font on lang install (someone correct me if I'm wrong), so even if we wanted that solution it's too invasive for v18.

**Comment by CvH** [comment]
> and supports CJK as far as I'm aware

I just looked into Noto (while searching an solution for LE) and there is a rather big size difference. 
Btw I am not trying to look like someone that has an idea about the font situation ;)

> I don't think we have the infrastructure in place

Yes likely there is nothing at K18 - the idea looks rather obvious to ship the needed font with the language addon. Also this would support all skins instead that every skin might need to ship their own one (not sure this is true or not).

**Comment by pkerling** [comment]
> I just looked into Noto (while searching an solution for LE) and there is a rather big size difference.

Which version did you investigate? Noto supports *a lot* of languages and quite thoroughly :-) I'm not sure we need that much. As said we could merge Latin characters into DejaVu to keep the current size.

**Comment by CvH** [comment]
![image](https://user-images.githubusercontent.com/1355173/50549425-08c30080-0c5d-11e9-90f3-25c10ff3e605.png)
https://fonts.google.com/specimen/Noto+Serif+SC?selection.family=Noto+Serif+SC

like said I have actually not really an idea about fonts :)

**Comment by CvH** [comment]
just to add, it is broken since at least K17, several other languages are broken too (maybe i missed some)
`amharic, arabic, armenian, burmese, hebrew, hindi, korean, persian , sinhala, tamil, telugu, thai`

**Comment by CastagnaIT** [comment]
same problem using netflix addon, whenever there are titles with languages as arabic or japanese or chinese etc... no symbols are show

example:
![image](https://user-images.githubusercontent.com/3257156/60394357-3eb94280-9b23-11e9-8e96-0e2661957d3e.png)

please update the skin font is incomprehensible and use arial based makes the screen pretty ugly

**Comment by JayXon** [comment]
Any update on this? Will font fallback be in v19?

**Comment by yol** [comment]
No one is currently working on this.
